scripts/ci/util.py

```diff
@@ -43,24 +42,28 @@ def _get_azext_metadata(ext_dir):
     return azext_metadata
 
 
-def get_ext_metadata(ext_dir, ext_file, ext_name):
-    # Modification of https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/extension.py#L89
-    WHL_METADATA_FILENAME = 'metadata.json'
+def get_ext_metadata(ext_dir, ext_file):
+    from pkginfo import Wheel
+
     zip_ref = zipfile.ZipFile(ext_file, 'r')
     zip_ref.extractall(ext_dir)
     zip_ref.close()
     metadata = {}
-    dist_info_dirs = [f for f in os.listdir(ext_dir) if f.endswith('.dist-info')]
     azext_metadata = _get_azext_metadata(ext_dir)
     if azext_metadata:
         metadata.update(azext_metadata)
-    for dist_info_dirname in dist_info_dirs:
-        parsed_dist_info_dir = WHEEL_INFO_RE(dist_info_dirname)
-        if parsed_dist_info_dir and parsed_dist_info_dir.groupdict().get('name') == ext_name.replace('-', '_'):
-            whl_metadata_filepath = os.path.join(ext_dir, dist_info_dirname, WHL_METADATA_FILENAME)
-            if os.path.isfile(whl_metadata_filepath):
-                with open(whl_metadata_filepath) as f:
-                    metadata.update(json.load(f))
+
+    try:
+        ext_wheel = Wheel(ext_file)
+
+        t = vars(ext_wheel)
+        del t['filename']
+        del t['description']    # del as description is trivial
+
+        metadata.update(t)
```

Comment:
I remember there was some example metadata shown, in another PR?

Reply:
The description is unnecessary and sometimes is very long, the content don't contribute to src/index.json. So I remove the property.